chsdi/views/stationboard.py

```diff
@@ -0,0 +1,85 @@
+# -*- coding: utf-8 -*-
+
+
+from sqlalchemy import and_
+from sqlalchemy.exc import OperationalError
+from pyramid.httpexceptions import HTTPInternalServerError, HTTPBadRequest
+from pyramid.view import view_config, view_defaults
+from chsdi.models.vector.uvek import OevDepartures
+import datetime
+from pytz import timezone
+
+
+@view_defaults(renderer='jsonp', route_name='stationboard')
+class TransportView(object):
+
+    def __init__(self, request):
+        self.request = request
+        if request.matched_route.name == 'stationboard':
+            id = request.matchdict['id']
+            try:
+                self.id = int(id)
+            except ValueError, e:
+                raise HTTPBadRequest(e)
+
+            self.destination = request.params.get('destination')
+
+    @view_config(request_method='GET')
+    def get_departures(self):
+        current_date = datetime.datetime.now(timezone('Europe/Berlin'))
```

Comment:
This is the time zone of the server not (time at which the request was performed)